app/components/settings/email-preferences-section.js

```diff
@@ -3,4 +3,9 @@ import Ember from 'ember';
 const { Component } = Ember;
 
 export default Component.extend({
+  actions: {
+    save(notificationType, setting) {
```

Comment:
@niranjan94 Directly use the action in route? or use ```sendAction method``` to redirect the it from component to my parent's controller?